cre/src/state_client.cpp

```diff
@@ -0,0 +1,77 @@
+// Copyright (c) 2018-2021 VMware, Inc. All Rights Reserved.
+//
+// This product is licensed to you under the Apache 2.0 license (the "License"). You may not use this product except in
+// compliance with the Apache 2.0 License.
+//
+// This product may include a number of subcomponents with separate copyright notices and license terms. Your use of
+// these subcomponents is subject to the terms and conditions of the subcomponent's license, as noted in the LICENSE
+// file.
+
+#include "state_client.hpp"
+#include "concord.cmf.hpp"
+#include "bftclient/quorums.h"
+
+namespace cre::state {
```

Comment:
why to have a namespace and a class with the same name?

Reply:
fixed